packages/console/src/pages/NotFound/index.module.scss

```diff
@@ -1,6 +1,11 @@
 @use '@/scss/underscore' as _;
 
 .container {
+  height: 100%;
+  padding: _.unit(6);
+}
+
+.content {
```

Comment:
just add a margin to the original container?

Reply:
It will cause the notFound to fulfill its parent, that's not what we want.
<img width="1920" alt="image" src="https://user-images.githubusercontent.com/10806653/223998336-c6b6aa65-c9f1-47e7-8989-8263fd414af3.png">

<img width="1907" alt="image" src="https://user-images.githubusercontent.com/10806653/223998565-e4c58deb-7141-4f5d-9595-978cc1050926.png">